src/macros.jl

```diff
@@ -625,12 +625,102 @@ julia> @with df @byrow :x * :y
     global scope, then a variable cannot be assigned without using the `global` keyword.
     If the parent scope is a local scope (inside a function or let block for example),
     the `global` keyword is not needed to assign to that parent scope.
+
+!!! note
+    Using `AsTable` inside `@with` block is currently not supported.
 """
 macro with(d, body)
     esc(with_helper(d, body))
 end
 
+astable_rhs_orderby_docs = """
```

Comment:
make it a `const` and uppercase the name?

Reply:
added.